src/components/utils/SaveButton.js

```diff
@@ -14,43 +13,45 @@ import {
     useMediaQuery,
     useTheme,
 } from "@material-ui/core";
-import ApplyIcon from "@material-ui/icons/Done";
+import SaveIcon from "@material-ui/icons/Save";
 
-const ApplyButton = (props) => {
-    const { baseId, applyDisabled, onApply } = props;
+const SaveButton = (props) => {
+    const { baseId, disabled, onSave, ...custom } = props;
 
     const { t } = useTranslation("common");
     const theme = useTheme();
     const isSmall = useMediaQuery(theme.breakpoints.down("sm"));
 
-    const buttonId = build(baseId, ids.BUTTONS.SAVE);
+    const buttonId = build(baseId, ids.SAVE_BTN);
 
     return isSmall ? (
-        <Tooltip title={t("apply")} placement="bottom" enterDelay={200}>
+        <Tooltip title={t("save")} placement="bottom" enterDelay={200}>
             <span>
                 <IconButton
                     id={buttonId}
-                    aria-label={t("apply")}
-                    disabled={applyDisabled}
-                    onClick={onApply}
+                    aria-label={t("save")}
+                    disabled={disabled}
+                    onClick={onSave}
                     color="primary"
+                    {...custom}
                 >
-                    <ApplyIcon />
+                    <SaveIcon />
                 </IconButton>
             </span>
         </Tooltip>
     ) : (
         <Button
             id={buttonId}
-            disabled={applyDisabled}
-            onClick={onApply}
+            disabled={disabled}
+            onClick={onSave}
             color="primary"
```

Comment:
Minor - May be `color` could also customized by `props` but default can be `primary`.

Reply:
It actually already works this way now 🎉 

Since all props except `baseId` and `onSave` are being passed along to the button in `{...custom}` (but this reminds me `disabled` will also get passed along and I don't have to set it explicitly).